components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -332,12 +343,12 @@ export const StyledUploadIconContainer = styled('div') <SelectionProps>`
   justify-content: center;
   gap: 16px;
   ${p => p.selected
-    ? css`    
-      background: ${p => isDarkTheme(p) ? '#525779' : '#F0F2FF'}; 
+    ? css`
+      background: ${p => isDarkTheme(p) ? '#525779' : '#F0F2FF'};
```

Comment:
lol, I thought you added these for a second and was like "where did these come from?"

Then I realised it was just formatting :D

Reply:
yeah, this was done automatically by presubmit fix.